datalad/__init__.py

```diff
@@ -16,6 +16,11 @@
 from .config import ConfigManager
 cfg = ConfigManager()
 
+from datalad.support.sshconnector import SSHManager
```

Comment:
or screw it for now --  seems to add only 1ms on subsequent runs.  I better remove import of numpy ;)